python/ray/workflow/workflow_access.py

```diff
@@ -205,136 +64,180 @@ def gen_step_id(self, workflow_id: str, step_name: str) -> str:
         else:
             return f"{step_name}_{idx}"
 
-    def _update_workflow_status(self, workflow_id: str, status: common.WorkflowStatus):
-        wf_store = workflow_storage.WorkflowStorage(workflow_id)
-        wf_store.update_workflow_status(status)
-        self._workflow_status[workflow_id] = status
-
-    def update_step_status(
+    def submit_workflow(
```

Comment:
Can we try to make  the private method starting with `_` for this class?

Reply:
this is a public method.